src/components/MarkdownProvider/index.tsx

```diff
@@ -48,12 +62,26 @@ export const MarkdownProvider: React.FC = ({ children }) => (
         CodeExample,
         PackageDescription,
         PropSheets,
-        h2: H2,
-        h3: H3,
-        h4: H4,
-        h5: H5,
-        h6: H6,
-        p: P
+        h2: StyledH2,
+        h3: StyledH3,
+        h4: StyledH4,
+        h5: StyledH5,
+        h6: StyledH6,
+        p: StyledP,
```

Comment:
Use the new Garden [`Paragraph`](https://garden.zendesk.com/react-components/typography/#paragraph-1) component (@hzhu will release today).

Reply:
Updated now that it is released.